hooks/mutations/delegation/useCancelSentRequestToBeDelegate.ts

```diff
@@ -0,0 +1,25 @@
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import { sentRequestsToBeDelegateKey } from "constants/queryKeys";
+import { useHandleError, useUserContext } from "hooks";
+import { DelegationEventT } from "types";
+import { removeDelegator } from "web3";
+
+export function useCancelSentRequestToBeDelegate() {
```

Comment:
This is used by a `pending-delegator` to cancel a pending setDelegate right?
I think it should renamed:
```suggestion
export function useCancelSentRequestToBeDelegator() {
```